contracts/rosca.sol

```diff
@@ -80,8 +82,8 @@ contract rosca {
         roundPeriodInDays = _roundPeriodInDays;
         if(roundSum < 1) throw; //if everyone is only required to contribute 0 , whats the point of rosca
```

Comment:
when i was writing this contract, i had "ether" in mind instead of "wei" for units, I think wei is a better unit to use